test/patest_suggested_vs_streaminfo_latency.py

```diff
@@ -29,7 +29,7 @@ def loadCsvData( dataFileName ):
     inputDevice = ""
     outputDevice = ""
 
-    startLines = file(dataFileName).readlines(1024)
+    startLines = open(dataFileName).readlines(1024)
```

Comment:
The `from pylab import *` was hiding the `file()` issue that is just like:
* #898